ci/citest/acceptance-test/multibox/10.0.100.14-vdc-executor-lxc/guestroot/etc/mesos-slave/attributes

```diff
@@ -1 +1 @@
-hypervisor:lxc
+hypervisor:lxc;node-groups:linuxbr
```

Comment:
Is it possible to have this on two lines?

```
hypervisor:lxc
node-groups:linuxbr
```

Also, is it possible to have multiple groups comma separated?

```
node-groups: linuxbr, foobar, hoge
```